packages/commonwealth/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM jnaviask/edgeware-node:v0.0.2
+FROM builder
 
-RUN apt-get install apt-transport-https
+COPY . .
 
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apk add --update --no-cache python3 build-base gcc && ln -sf \
+/usr/bin/python3 /usr/bin/python && apk add postgresql-client
 
-RUN apt-get update -y
+ENV WAIT_VERSION 2.7.2
 
-# for puppeteer
-RUN apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
-libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
-libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
-libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \
-ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget git
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/$WAIT_VERSION/wait /wait
+
+RUN chmod +x /wait
+
+RUN yarn --ignore-engines && apk add git && yarn build
```

Comment:
We only need to `yarn build` for production -- `yarn start` runs ts-node-dev which doesn't require compilation + picks up filesystem changes.